### DISCUSSION TITLE
Project description: "backup solution" still up2date?

### DISCUSSION THREAD

**Discussion by cswrd** [immich-app/immich#8019] [General]
According to the subtitle in the README immich is a _High performance self-hosted photo and video backup solution_.

According to the subtitle in the [official docs](https://immich.app/docs/overview/introduction) its a _management solution_.

I almost skipped this awesome project because I read the 1st description and thought its actually just that. I guess the projects is aiming towards _management solution_ or photo & video album media server?

**Comment by alextran1502** [comment]
We are in the progress of updating the logo/tagline across the sites. Thanks for the reminder
